.github/workflows/ci.yml

```diff
@@ -0,0 +1,126 @@
+---
+name: CI
+on:
+  push:
+    paths:
+      - "Formula/*.rb"
+    branches:
+      - master
+
+  pull_request:
+    paths:
+      - "Formula/*.rb"
+    branches:
+      - master
+
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test Formulae
+    runs-on: macos-11
```

Comment:
Can we test on multiple vms? like this:
https://github.com/Homebrew/homebrew-core/blob/6287ae3017b91a518347e69c8587aa9e9b6d5a64/.github/workflows/tests.yml#L183-L191

Reply:
We can.. but would take some extra work I don't want to put into this repo just yet.

That would be accomplished via GH actions' `matrix`/`strategy` syntax: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-running-multiple-versions-of-nodejs

The list of platforms available are here: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Notably lacking are Mac OS arm64/M1 processor architecture support. See also: https://github.com/actions/virtual-environments/issues/2187